src/core/constants.mjs

```diff
@@ -25,7 +25,7 @@ export const CUR_FILE_DIR = dirname(fileURLToPath(import.meta.url))
 export const USER = `${process.env.USER}`
 export const USER_SANITIZED = USER.replace(/[\W_]+/g, '-')
 export const SOLO_HOME_DIR = process.env.SOLO_HOME || `${process.env.HOME}/.solo`
-export const SOLO_LOGS_DIR = `${SOLO_HOME_DIR}/logs`
+export const SOLO_LOGS_DIR = `${SOLO_HOME_DIR}/output`
```

Comment:
Since solo logs folder path is changed, you will need to update the DEV.md instructions accordingly.